docs/rules/assumed-size-character-intent.md

```diff
@@ -14,24 +14,24 @@ program example
 contains
   subroutine set_text(text)
     character(*), intent(out) :: text
-    text = \"longer than 3 characters\"
+    text = "longer than 3 characters"
   end subroutine set_text
 end program
 ```
 
-Here, `short_text` will only contain the truncated \"lon\".
+Here, `short_text` will only contain the truncated "lon".
 
 To handle dynamically setting `character` sizes, use `allocatable` instead:
 
 ```f90
 program example
-  character(len=3) :: short_text
-  call set_text(short_text)
-  print*, short_text
+  character(len=:), allocatable :: allocatable_text
```

Comment:
I am not sure what would be a good name for this variable.

Reply:
I think this is fine